dialogue-core/src/main/java/com/palantir/dialogue/core/BalancedNodeSelectionStrategyChannel.java

```diff
@@ -56,13 +68,24 @@
 final class BalancedNodeSelectionStrategyChannel implements LimitedChannel {
```

Comment:
I think we should update the description so that it includes the additional parameters to our cost function